libs/application/templates/new-primary-school/src/forms/Prerequisites.ts

```diff
@@ -82,13 +82,28 @@ export const Prerequisites: Form = buildForm({
             buildDescriptionField({
               id: 'noChildrenNationalRegistryDescription',
               title: '',
-              marginTop: 5,
+              marginTop: 3,
+              description:
+                newPrimarySchoolMessages.pre
+                  .noChildrenFoundNationalRegistryDescription,
+            }),
+            buildDescriptionField({
+              id: 'noChildrenIslandisInfo',
+              title: '',
+              marginTop: 2,
+              description:
+                newPrimarySchoolMessages.pre.noChildrenFoundIslandisDescription,
+            }),
+            buildDescriptionField({
+              id: 'noChildrenNationalRegistryInfo',
+              title: '',
+              marginTop: 3,
               description:
                 newPrimarySchoolMessages.pre
                   .noChildrenFoundNationalRegistryInfo,
             }),
             buildDescriptionField({
-              id: 'noChildrenMMSDescription',
+              id: 'noChildrenMMSInfo',
               title: '',
               marginTop: 3,
               description: newPrimarySchoolMessages.pre.noChildrenFoundMMSInfo,
```

Comment:
There is no Margin at the bottom of the page, add `marginBottom: 6`
![image](https://github.com/island-is/island.is/assets/54938148/897a1cea-d58b-4c5b-83d4-f508b6915ac8)